tests/queue/deadlock_symetrical.cpp

```diff
@@ -0,0 +1,73 @@
+/* RUN: %{execute}%s
+
+   Test that there is no deadlock in with massive parallel launch of
+   kernels using accessors in an asymmetrical creation, A B || B A.
```

Comment:
should the name of this test be deadlock_asymmetrical.cpp rather than deadlock_symmetrical.cpp? Or are the comments for deadlock_symetrical.cpp and deadlock.cpp in reverse? I believe it's the former if something is incorrect